hs_discover/src/components/Resources.vue

```diff
@@ -0,0 +1,752 @@
+                        </td>
+                        <!-- python is passing .isoformat() in views.py -->
+                        <td style="cursor:pointer" data-toggle="tooltip" :title="new Date(entry.created).toLocaleTimeString('en-US')">{{new Date(entry.created).toLocaleDateString('en-US')}}</td>
+                        <td style="cursor:pointer" data-toggle="tooltip" :title="new Date(entry.created).toLocaleTimeString('en-US')">{{new Date(entry.modified).toLocaleDateString('en-US')}}</td>
+                    </tr>
+                    </tbody>
+                </table>
+            </div>
+        </div>
+    </div>
+    </div>
+</template>
+
+<script>
+import DatePick from 'vue-date-pick';
+import 'vue-date-pick/dist/vueDatePick.css';
+import axios from 'axios'; // css font-size overridden in hs_discover/index.html to enforce 1em
+
+export default {
+  data() {
+    return {
+      resloaded: false,
+      resources: [],
+      searchtext: '',
+      filterlimit: 10, // Minimum threshold for filter item to display with checkbox
+      geodata: [],
+      geopoints: [],
+      startdate: 'Start Date',
+      enddate: 'End Date',
+      filteredcount: 0,
+      perpage: 40, // resources per page
+      pagenum: 1, // initial page number to show
+      geoloaded: false, // searchjson endpoint called and retrieved geo data
+      resgeotypes: '',
+      googMarkers: [],
+      uidFilter: [],
+      countAuthors: {},
+      authorFilter: [],
+      countSubjects: {},
+      subjectFilter: [],
+      countOwners: {},
+      ownerFilter: [],
+      countContributors: {},
+      contributorFilter: [],
+      countTypes: {},
+      typeFilter: [],
+      countAvailabilities: {},
+      availabilityFilter: [],
+      sortDir: -1, // 1 asc -1 desc
+      sortingBy: 'modified',
+      resIconName: {
+        'Composite Resource': '/static/img/resource-icons/composite48x48.png',
+        Generic: '/static/img/resource-icons/generic48x48.png',
+        'Geopgraphic Raster': '/static/img/resource-icons/geographicraster48x48.png',
+        'Model Program Resource': '/static/img/resource-icons/modelprogram48x48.png',
+        'Collection Resource': '/static/img/resource-icons/collection48x48.png',
+        'Web App Resource': '/static/img/resource-icons/webapp48x48.png',
+        'Time Series': '/static/img/resource-icons/timeseries48x48.png',
+        'Script Resource': '/static/img/resource-icons/script48x48.png',
+        'Model Instance Resource': '/static/img/resource-icons/modelinstance48x48.png',
+        'SWAT Model Instance Resource': '/static/img/resource-icons/swat48x48.png',
+        'MODFLOW Model Instance Resource': '/static/img/resource-icons/modflow48x48.png',
+        'Multidimensional (NetCDF)': '/static/img/resource-icons/multidimensional48x48.png',
+        'HIS Referenced Time Series': '/static/img/resource-icons/his48x48.png',
+      },
+      sortMap: {
+        'First Author': 'author',
```

Comment:
These keys are inconsistent with some using a string for the key and others not.

Reply:
If you're referring to Generic, that's literally an eslint standard. I was surprised by it, too.

Reply:
Same for Owner, Creator...

Reply:
Weird, any way we can just enforce always using quotes?  I'd imagine the quotes are necessary because of the space in the key.

Reply:
As it is functioning and per the syntax specification it should stay as written.